tests/usecase/test_emit_metrics.py

```diff
@@ -1,8 +1,25 @@
 import basetest
+import copy
 import time
 
+from unittest import TestCase
+from unittest.mock import Mock
+
+from metrics import FreeAppsMetricsEmitterThread
```

Comment:
shouldn't this actually be `from lib.metrics import blah`?

I suspect that this only works as a side-effect of some of the horrible path manipulation things

Reply:
True, it's horrible path manipulation on all levels. This is exactly why I wouldn't break the pattern now because that might have some unforseen consequences that could take a long and tedious time to fix, or just simply adding more to the "snowflakiness".